osbrain/agent.py

```diff
@@ -1598,7 +1598,7 @@ def send_recv(self, address, message):
     @Pyro4.oneway
     def run(self):
         """
-        Begin execution of the agent process and start the main loop.
+        Start the main loop of the agent process.
```

Comment:
I would not use the word process here. Although it is usually the case, it does not necessarily need to be like that. An agent could run on a thread, or you could even run its main loop from `__main__` if you wanted to. As you see, `Agent` does not inherit from `multiprocessing.Process`.

So I would rather say "Start the main loop of the agent." or "Start the agent's main loop".